router.js

```diff
@@ -23,145 +24,135 @@ const repositorySchema = Joi.object({
     language: Joi.string()
 });
 
-router.get('/hello', (ctx) => ctx.body = 'Hello World !');
+router.get('/hello', (req, res) => res.send('Hello World !'));
 
 // Repository
-router.get('/repository/:id', async (ctx) => {
-    const { value, error } = Joi.attempt(ctx.params.id, Joi.number().integer());
-    if (error) {
-        ctx.body = error.message;
-        ctx.status = 403;
-        return;
+router.get('/repository/:id', async (req, res) => {
+    const id = req.params.id;
+    try {
+        Joi.attempt(id, Joi.number().integer());
+    } catch (error) {
+        res.status(403).send(error.message);
     }
-
     try {
-        ctx.body = await repository.read({ id: value });
+        response = await repository.read({ id });
+        res.json(response);
     } catch {
-        ctx.status = 500;
+        res.status(500).end();
     }
 });
-router.get('/repository', async (ctx) => {
-    const { value, error } = repositorySchema.validate(ctx.query);
+
+router.get('/repository', async (req, res) => {
+    const { value, error } = repositorySchema.validate(req.query);
     if (error) {
-        ctx.body = error.message;
-        ctx.status = 403;
-        return;
+        res.status(403).send(error.message);
     }
     try {
-        ctx.body = await repository.read(value);
+        const response = await repository.read(value);
+        res.json(response);
     } catch {
```

Comment:
Catch should pass on the error to at least a logger, or to the res function, or an error handler middleware. `res.status` is not enough in it self, please use `res.sendStatus()`.